### PR TITLE
Fix issue 373

### PR DESCRIPTION
**Pull request by mmoerdijk**
### Description
Added the set_max_acceleration_scaling_factor function to the python api moveit commander. Should be cherry-picked to Kinetic. 

### Checklist
- [ ] **Required**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Optional: Decide if this should be cherry-picked to other current ROS branches (Indigo, Jade, Kinetic)

Thank you!

### CODE REVIEW

**Comment by rbbg** [comment]
Looks good :) just remove the line (#585) in `wrap_python_move_group.cpp` and clang-format will also pass.

File: moveit_commander/src/moveit_commander/move_group.py [hunk 1]
```diff
@@ -409,6 +409,14 @@ def set_max_velocity_scaling_factor(self, value):
         else:
             raise MoveItCommanderException("Expected value in the range from 0 to 1 for scaling factor" )
 
+
```
Comment (davetcoleman) [inline moveit_commander/src/moveit_commander/move_group.py]: extra blank line?

Comment (mmoerdijk) [inline moveit_commander/src/moveit_commander/move_group.py]: Removed! Missed it somehow

**Comment by mmoerdijk** [comment]
Evidently running clang format is not without risks I got errors afterwards when compiling. Clang format messed up a nested template argument. Is this normal? I'm using clang-format-3.6 in combination with the config described [here](https://github.com/ros-industrial/ros_qtc_plugin/wiki/3.-Setup-Qt-Creator-for-ROS#section3.4)

**Comment by davetcoleman** [comment]
Our clang-format is probably different than ROS Industrial's. Our style is located in the root of the repo (.clang-format) and instructions are in the pull request template, which links to http://moveit.ros.org/documentation/contributing/code/

It should never cause a compiler error, but perhaps the version you used caused ``> >`` to become ``>>`` which is invalid before C++11

**Comment by davetcoleman** [comment]
ping - are you still having issues with clang-format?

**Comment by davetcoleman** [comment]
Looks like my previous example is exactly what is breaking the build now:

```
error: ‘>>’ should be ‘> >’ within a nested template argument list
```

**Comment by mmoerdijk** [comment]
@davetcoleman Finally it worked out. For me the Christmas Holidays already started and I don't have my setup at home to test it.

**Comment by v4hn** [comment]
> For me the Christmas Holidays already started and I don't have my setup at home to test it.

Happy Holidays :)

Does that comment mean we should wait until you get back to your setup in January to test it?
Is this ready to merge from your point of view @mmoerdijk ?

**Comment by davetcoleman** [comment]
I was going to merge this but wasn't at a terminal to cherry-pick to Kinetic at the same time. I am now, and this appears super benign

**Pull request merged**